src/ProjectNeon/Assets/Data/Templates/Deck.cs

```diff
@@ -5,8 +5,14 @@ public class Deck : ScriptableObject
 {
     [SerializeField]
     private List<Card> cards;
+
     public void Shuffle()
     {
     }
 
+    public List<Card> GetCards()
```

Comment:
Turn this into a public Property, and return a copy of the cards `.ToList()`, so that any mutation won't change the Asset file.

Reply:
@SilasReinagel Done, couldn't find `.ToList()` but i've copied the list by hand